### PR TITLE
OSDOCS#8971: Docs for change in secret generation behavior when the i…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…nternal registry is disabled

Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-8971

Link to docs preview:
* https://71256--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/cluster-capabilities#cluster-image-registry-operator_cluster-capabilities
* https://71256--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/pods/nodes-pods-secrets#auto-generated-sa-token-secrets_nodes-pods-secrets

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/default-secrets.adoc
Comment (bergerhoffer) [inline modules/default-secrets.adoc:22]: @sanchezl Could there be more secrets that are included in this list depending on other add-ons? If so, should we mention that, or leave it be?

File: modules/default-secrets.adoc
Comment (bergerhoffer) [inline modules/default-secrets.adoc:18]: @sanchezl Is the general statement below about these secrets being used to access the internal registry good, or is there some more info we could expand that with? Or give more detail for each of these 6 specifically?

Comment (sanchezl) [reply modules/default-secrets.adoc:18]: This needs to be a general statement, I provided an alternate text suggestion in another comment.

File: modules/default-secrets.adoc
Comment (sanchezl) [reply modules/default-secrets.adoc:22]: There are hundreds.

File: modules/service-account-auto-secret-removed.adoc
Comment (sanchezl) [inline modules/service-account-auto-secret-removed.adoc:14]: The deleted text still applies. I would keep the NOTE and add the IMPORTANT section.

**Review by sanchezl — CHANGES_REQUESTED** [review]

File: modules/cluster-image-registry-operator.adoc
Comment (bergerhoffer) [reply modules/cluster-image-registry-operator.adoc:48]: Updated, slightly modified

**Review by sanchezl — APPROVED** [review]

**Comment by smunje1** [comment]
@bergerhoffer This changes LGTM. As I am new to the peer review squad, I request that you wait until @bscott-rh gives approval.

File: modules/cluster-image-registry-operator.adoc
Comment (bscott-rh) [inline modules/cluster-image-registry-operator.adoc:36]:
```suggestion

```
As discussed, this will be removed. The rest of the changes LGTM.

Comment (bergerhoffer) [reply modules/cluster-image-registry-operator.adoc:36]: Yes thanks :woman_facepalming: thanks for catching!

**Pull request merged**
